config/default.cfg

```diff
@@ -951,6 +952,9 @@ cfg$gms$c_BaselineAgriEmiRed <- 0
 #  (3) upper estimate: 0.075
 #  (4) unconstrained: 1
 #  (5) sustainability case: 0.001
+# c_ccsinjecrateRegi  "regional upper bound of the CCS injection rate, overwrites settings set with c_ccsinjectratescen"
+#  ("off") no regional differentiation
+#  ("GLO 0.000625, CAZ_regi 0.00225, CHA_regi 0.002, EUR_regi 0.00225, IND_regi 0.002, JPN_regi 0.001, USA_regi 0.001"): "example that is taylored such that NDC goals are achieved without excessive CCS in a delayed transition scenario with c_ccsinjecratescen = 1. Globally, 75% reduction, 10% reduction in CAZ etc."
```

Comment:
This comments sounds like the default value would be 0.0025, but that is the lower estimate. The default is 0.005.